src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -325,6 +347,48 @@ private void generateIntervalsVCFFileFromAllShards() {
         intervalsVCFWriter.close();
     }
 
+    private void concatenateDenoisedCopyRatioFiles() {
+        if (outputDenoisedCopyRatioFile != null) {
+            logger.info("Concatenating the denoised copy ratios...");
```

Comment:
To be consistent with the log messages for VCF output, should be `logger.info("Generating denoised copy ratios...")`.

Reply:
Done